### PR TITLE
Update README.md and fix yml language naming

### PR DESCRIPTION
**Pull request by rdimitrov**
Signed-off-by: Radoslav Dimitrov <rdimitrow@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,9 @@ $ faas new --list
 
 Languages available as templates:
 - golang-http
+- golang-http-armhf
 - golang-middleware
+- golang-middleware-armhf
```
Comment (alexellis) [inline README.md:14]: Please add a short sentence or paragraph explaining which to use for ARMv7 or Raspberry Pi.

File: template/golang-http/template.yml [hunk 2]
```diff
@@ -1,2 +1,2 @@
-language: go
+language: golang-http
```
Comment (alexellis) [inline template/golang-http/template.yml:1]: lgtm

File: README.md (hunk 1 above)
Comment (rdimitrov) [reply README.md:14]: Got it :+1:

Comment (rdimitrov) [reply README.md:14]: Addressed it

**Review by alexellis — APPROVED** [review]
LGTM

**Pull request merged**